images/converter/default.go

```diff
@@ -149,7 +149,7 @@ func (c *defaultConverter) convertLayer(ctx context.Context, cs content.Store, d
 // - records diff ID changes in c.diffIDMap
```

Comment:
> // - clears `.mediaType` if the target format is OCI

Now we shouldn't

Reply:
(But maybe we should still have an option to clear media type, for potential compatibility issue. )